lib/rubocop/result_cache.rb

```diff
@@ -128,7 +128,8 @@ def rubocop_checksum
         begin
           lib_root = File.join(File.dirname(__FILE__), '..')
           bin_root = File.join(lib_root, '..', 'bin')
-          source = Find.find(lib_root, bin_root).sort.map do |path|
+          source_files = $LOADED_FEATURES + Find.find(lib_root, bin_root).to_a
```

Comment:
It's a very simple way of finding all the source files that can affect RuboCop's behavior, which is why I suggested it in #2886. I don't think it affects the start-up time severely in the normal case (a reasonably performant file system). The files under `lib` will be included in `$LOADED_FEATURES` BTW, so you can remove `lib_root`.